scripts/upgrade

```diff
@@ -158,6 +160,23 @@ ynh_script_progression --message="Upgrading NGINX web server configuration..." -
 # Create a dedicated NGINX config
 ynh_add_nginx_config
 
+#=================================================
+# UPGRADE DEPENDENCIES
+#=================================================
+ynh_script_progression --message="Upgrading dependencies..." --weight=10
+
+ynh_exec_warn_less ynh_install_app_dependencies $pkg_dependencies
```

Comment:
Will that work is postgres is updated to a new major version, and needs migrating the data to a new format?

Reply:
I think yes, postgresql migration is managed by yunohost migration